src/components/Project/Project.types.ts

```diff
@@ -0,0 +1,14 @@
+interface Link {
```

Comment:
1) Is there a reason why this can't be in: src -> dataModels?
2) This starts the convention of using .types. I am not oppose to it.

Reply:
> 1. Is there a reason why this can't be in: src -> dataModels?

Nope! Admittedly though, I'm not super familiar with this structure, is what I did in [14118e8](https://github.com/OpenBCca/openbc-web/pull/5/commits/14118e8f39ad5875fd2e47842510401159908643) what you had in mind?

Reply:
We are planning on putting models into the following folder structure

src/dataModels.

So I prefer two files.

One for link and another for Projects.

![image](https://github.com/OpenBCca/openbc-web/assets/40151580/98d03bd2-2884-4b26-bc9f-d43b5c2b39f5)

Reply:
Addressed in [d770d98](https://github.com/OpenBCca/openbc-web/pull/5/commits/d770d98659791cc89e718aeaf77ae8f12d09a5ab), thanks for the additional context @SamHuo213